defaults/main.yml

```diff
@@ -10,3 +10,4 @@ mas_installed_apps:
 
 mas_upgrade_all_apps: false
 mas_signin_dialog: false
+mas_binary: /usr/local/bin/mas
```

Comment:
Hardcoding this path makes the role more tied to specific versions of mas/installation methods that put it in the local bin path (whereas people may have it installed elsewhere...). Maybe keep the default `mas` so it doesn't break any existing usage?

Reply:
Just opened https://github.com/geerlingguy/ansible-role-homebrew/issues/156 and homebrew on ARM recommends `/opt/homebrew` as `/usr/local/bin` is default for Intel so "already" a problem.

Do you mean do something like this?
```yaml
- name: Sign in to MAS when email and password are provided.
  command: '{{ mas_binary | default("mas") }} signin "{{ mas_email }}" "{{ mas_password }}"'
  register: mas_signin_result
  when:
    - mas_account_result.rc == 1
    - mas_email | bool
    - mas_password | bool
    - not mas_signin_dialog
 ```